pkg/profiler/cpu/bpf/maps/maps.go

```diff
@@ -52,15 +52,14 @@ import (
 )
 
 const (
-	DebugThreadsIDsMapName        = "debug_threads_ids"
-	StackCountsMapName            = "stack_counts"
-	eventsCountMapName            = "events_count"
-	StackTracesMapName            = "stack_traces"
-	HeapMapName                   = "heap"
-	InterpreterStackTracesMapName = "interpreter_stack_traces"
-	SymbolIndexStorageMapName     = "symbol_index_storage"
-	SymbolTableMapName            = "symbol_table"
-	eventsMapName                 = "events"
+	DebugThreadsIDsMapName    = "debug_threads_ids"
```

Comment:
(not the concern of this PR)
nit: Why do we export some of the constants and some of them not?

Reply:
Good shout, we use some of these constants in other packages, but let's not export the ones that don't need to be public